internal/portforward/loop.go

```diff
@@ -157,6 +157,20 @@ func (l *Loop) GetPortsForwarded() (ports []uint16) {
 	return l.service.GetPortsForwarded()
 }
 
+func (l *Loop) SetPortsForwarded(ports []uint16) (err error) {
+	if l.service == nil {
+		return
+	}
+
+	err = l.service.SetPortsForwarded(l.runCtx, ports)
+	if err != nil {
+		l.logger.Error(err.Error())
+		return err
```

Comment:
Let's remove the logs in the port forwarding code and let the calling layers log out the error if necessary (in this case in the control server code) - sorry if I might had changed my mind on this!

```suggestion
	if err != nil {
		return err
```

Reply:
No worries, see e9aaa9712f2f2fcadf21523e5ce0466fc6f2b5c0